src/components/AppHeader.tsx

```diff
@@ -43,7 +51,7 @@ export const AppHeader = ({ appName, appIcon, userSession, open }) => (
               <UserMenu
                 alignSelf="center"
                 user={userSession.user}
-                items={userSession.items}
+                items={userSession.items as []}
```

Comment:
this should not be necessary (please improve the interface - see previous comment)